proxmoxtf/resource/container.go

```diff
@@ -1216,35 +1225,11 @@ func containerCreateCustom(ctx context.Context, d *schema.ResourceData, m interf
 		}
 	}
 
-	featuresBlock, err := structure.GetSchemaBlock(
-		resource,
-		d,
-		[]string{mkResourceVirtualEnvironmentContainerFeatures},
-		0,
-		true,
-	)
+	features, err := containerGetFeatures(resource, d)
 	if err != nil {
 		return diag.FromErr(err)
 	}
 
-	nesting := types.CustomBool(featuresBlock[mkResourceVirtualEnvironmentContainerFeaturesNesting].(bool))
-	keyctl := types.CustomBool(featuresBlock[mkResourceVirtualEnvironmentContainerFeaturesKeyControl].(bool))
-	fuse := types.CustomBool(featuresBlock[mkResourceVirtualEnvironmentContainerFeaturesFUSE].(bool))
-
-	features := containers.CustomFeatures{}
-
-	if nesting {
-		features.Nesting = &nesting
-	}
-
-	if keyctl {
-		features.KeyControl = &keyctl
-	}
-
-	if fuse {
-		features.FUSE = &fuse
-	}
```

Comment:
these checks are necessary to avoid zeroing flags that are not updated.

e.g.
```hcl
  features {
    mount = ["nfs", "cifs"]
  }
```
produces this now:
<img width="501" alt="Screenshot 2023-12-06 at 10 58 26 AM" src="https://github.com/bpg/terraform-provider-proxmox/assets/627562/87bf06cd-8450-405f-ae2d-4036fb08e753">

which is not terrible, but better to be avoided.